components/AppBar/examples/AppBarImageryExample.m

```diff
@@ -28,7 +28,11 @@ - (void)viewDidLoad {
   [super viewDidLoad];
 
   // Create our custom image view and add it to the header view.
-  UIImageView *imageView = [[UIImageView alloc] initWithImage:[self headerBackgroundImage]];
+  UIImageView *imageView = [[UIImageView alloc]
```

Comment:
Super-nit: two-liner?

```
UIImage ...
UIImageView ...
```

Reply:
Clang-format!

Reply:
It's actually a super long line and it's breaking it up at each opportunity.

Reply:
Yah, I know. Robots will be robots...  Feel free not to fix. 👍